components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -41,6 +40,9 @@ type Props = {
     creatorName: string;
     teammate: UserProfileRedux;
     teammateName?: string;
+    stats: any;
+    usersLimit: number;
+    isCloud: boolean;
```

Comment:
I believe the cloud check is no longer needed, given our discussions with Eric S.